integration-tests/src/test/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG ALFRESCO_BASE_IMAGE
```

Comment:
This file is not used, because you are using the `docker-alfresco` gradle plugin